### ISSUE TITLE
Installer issues both Aut and Man

### ISSUE THREAD

**Issue by Matthuffy** [therecluse26/PHP-Login#201]
Both Manual and Auto install has issues and will fail. 
Manual has database errors to copy 
Auto has issues generating the dbconf.php file.

To fix I had to run first manual but did not manually copy the database. I ran the composer though.
I copied and created the dbconf.php

then I went back and ran the Auto installer and let it fail with a warning that something had already been initiated. 

Then I went to index.php and all was then good.

the installers have issues that need to be addressed.

**Comment by Matthuffy** [comment]
Then on top of that
editconfigajax.php is missing but required to make any edits..

**Comment by therecluse26** [comment]
Yeah, I know they have issues. I need to overhaul the installation process, and I'm working on a cleaner way of handling database migrations/seeding at the moment. That doesn't explain why you're even seeing a reference to `editconfigajax.php` though. Where did you download this from? That's an old name for the `config_updatesettings.php` ajax page.

**Comment by Matthuffy** [comment]
Yeah, sorry the Ajax error was only down to cache. Did not have time to update to let you know was my end not the script. Once cleared cache all was good. But the installer has a lot of errors. My work around works for now as I did not have the time to update your script I tried that solution and thankfully worked.

**Comment by Matthuffy** [comment]
I already have a decent installer script that I created, would work well for this. Could implement and give an update if you like but won't be able to for around a week as chaos here

**Comment by therecluse26** [comment]
@Matthuffy That would be amazing, honestly. I'm crazy busy here as well, and definitely want to revamp the install process, but I have a bunch of other things I'm working on on top of this, so I don't have tons of time. I'm currently playing around with integrating proper database migrations using Phinx in the dev branch, to sort of unify the database creation process and eliminate all of the gross procedural crap code I have in the installer currently. That would also make things easier with updating as well, I'm thinking, as they could just use the migrations to bring their databases into the current state as changes to the db roll in. Just a thought. Thanks man

**Comment by therecluse26** [comment]
@Matthuffy Do you have the installer script on github? I'd love to poke around and see how it works

**Comment by Matthuffy** [comment]
No, but will put it on in the next couple of days. Actually will be included in a script I will add tomorrow but will put the installer as stand alone so you can see it.

**Comment by Matthuffy** [comment]
Never bothered to make it useful for others as a plug and play of sorts so never bothered to add here, I just use for things I create for others

**Comment by therecluse26** [comment]
Well that's awesome. It would be greatly appreciated. I'm wrestling right now with getting migrations to work properly, but hopefully that will help add some consistency to the process as well. There really should be a standardized method of doing installers/updaters for web applications. I haven't been able to find much of anything about it. It seems kind of ad-hoc, in general.
